Plugins/Statistics/GitStatistics/PieChart/ColorUtil.cs

```diff
@@ -33,7 +33,7 @@ public static class ColorUtil
         /// </returns>
         public static Color CreateColorWithCorrectedLightness(Color color, float correctionFactor)
         {
-            Debug.Assert(correctionFactor <= 1 && correctionFactor >= -1, "correctionFactor <= 1 && correctionFactor >= -1");
+            Debug.Assert(correctionFactor is <= 1 and >= -1, "correctionFactor <= 1 && correctionFactor >= -1");
```

Comment:
nit:
```suggestion
            Debug.Assert(correctionFactor is <= 1 and >= -1, "correctionFactor is <= 1 and >= -1");
```

Reply:
If not necessary, omit the parens. `(2 * 3) + 4` is overkill to me - but required.

Reply:
I'd make it more human readable :)


```suggestion
            Debug.Assert(correctionFactor is <= 1 and >= -1, "correctionFactor is expected to be in [-1, 1] range");
```